src/test/java/persistence/sql/ddl/QueryGeneratorTest.java

```diff
@@ -0,0 +1,28 @@
+package persistence.sql.ddl;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class QueryGeneratorTest {
+    @Test
+    void create() {
+        final QueryGenerator queryGenerator = new QueryGenerator();
+        assertEquals(expected(), queryGenerator.create(Person.class));
+    }
```

Comment:
`@DisplayName`을 통해 조금 더 가독성있는 테스트를 만들어보면 좋을 것 같습니다! :)

```suggestion
    @Test
    @DisplayName("클래스 정보를 통해 CREATE 쿼리를 생성한다")
    void create() {
        final QueryGenerator queryGenerator = new QueryGenerator();
        assertEquals(expected(), queryGenerator.create(Person.class));
    }
```